system_tests/bigquery.py

```diff
@@ -602,3 +613,18 @@ def test_dump_table_w_public_data(self):
         dataset = Config.CLIENT.dataset(DATASET_NAME, project=PUBLIC)
         table = dataset.table(TABLE_NAME)
         self._fetch_single_page(table)
+
+    def test_create_table_nested_schema(self):
+        TABLE_NAME = 'test_table'
```

Comment:
I agree with @dhermes here - "constants" in Python use are module-level or class-level things that are *intended* to be immutable. If we start making stuff within a function scope a constant we might as well just turn on caps-lock and be done with it.